Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5232,35 +5451,135 @@ def panorama_download_latest_content_update_content(target: str):
     return result
 
 
-def panorama_download_latest_content_update_command(target: Optional[str] = None):
+def panorama_download_latest_content_update_command(args):
     """
     Download content and show message in war room
     """
+    target = args.get('target')
+    use_polling = args.get('polling', 'false') == 'true'
+    job_id = args.get('job_id')
+    timeout = int(args.get('timeout', 600))
+    interval_in_seconds = int(args.get('interval_in_seconds', 60))
+    cmd = demisto.command()
```

Comment:
same comments as above, use the build in ```CommonServerPython``` function to parse from string to int/bool